cypress/e2e/search.cy.ts

```diff
@@ -0,0 +1,40 @@
+import { NAVIGATION } from "../elements/navigation";
+import { SEARCH_PAGE_SELECTORS } from "../elements/search-page";
+
+describe("Search for products", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+  it("should search for products SRS_0405", () => {
+    const typedText = "polo";
+    cy.get(NAVIGATION.searchIcon)
```

Comment:
Could we change this:
```
 cy.get(NAVIGATION.searchIcon)
      .click()
      .get(SEARCH_PAGE_SELECTORS.searchInput)
      .type(typedText, { delay: 100 })
      .should("have.value", typedText)
```      
To one function and reuse it, because this code is same as in SRS_0405?

Reply:
I added navigateAndSearch function